llm/emergency_parser.py

```
@@ -0,0 +1,122 @@
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
+
+from dotenv import load_dotenv
+from openai import OpenAI
+from config import LLM_API_KEY
+
+# 환경 변수 로드
+load_dotenv()
+
+if not LLM_API_KEY:
+    raise RuntimeError("GEMMA_API_KEY not found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)